tests/share/test_config.py

```diff
@@ -727,3 +843,4 @@ def test_parse_config(self) -> None:
             assert elasticsearch.api_key == "api_key"
             assert elasticsearch.dataset == "dataset"
             assert elasticsearch.namespace == "namespace"
+            assert elasticsearch.tags == ["2021", "{'key1': 'value1'}", "['elem1', 'elem2']"]
```

Comment:
good test case. but actually we should reject it :)
`tags` must be `list[str]`

Reply:
should we reject every tag? let's say the user inputs `"tag1", 2, "tag3"`. We're dropping all tags or only the `2`?

Reply:
I would reject everything: it's a validation error.

Btw: we parse the config in the handler and use `Output` for the shipper factory, we should add a bad path test on `tests.handlers.aws.test_handler.TestLambdaHandlerFailure.test_lambda_handler_failure` for tags failing validation.

we should have done for the secret manager as well, I will open a ticket for it :)